src/pages/search/SearchResults.tsx

```diff
@@ -30,29 +34,37 @@ const SearchResults: FC<SearchResultsProps> = ({ publications, searchTerm }) =>
       {searchTerm && t('results', { count: publications.length, term: searchTerm })}
       <List>
         {publications &&
-          publications.slice(validPage * rowsPerPage, validPage * rowsPerPage + rowsPerPage).map((publication) => (
-            <PublicationListItemComponent
-              key={publication.identifier}
-              primaryComponent={
-                <MuiLink component={Link} to={`/publication/${publication.identifier}/public`}>
-                  {publication.mainTitle}
-                </MuiLink>
-              }
-              secondaryComponent={
-                <Typography component="span">
-                  {new Date(publication.modifiedDate).toLocaleDateString()}
-                  <br />
-                  {publication.contributors.map((contributor) => (
-                    <Fragment key={contributor.identifier}>
-                      <MuiLink component={Link} to={`/user/${contributor.identifier}`}>
-                        {contributor.name}
-                      </MuiLink>{' '}
-                    </Fragment>
-                  ))}
-                </Typography>
-              }
-            />
-          ))}
+          publications.slice(validPage * rowsPerPage, validPage * rowsPerPage + rowsPerPage).map((publication) => {
+            const publicationId = publication.id?.split('/').pop();
+            const displayDate = publication.date && new Date(publication.date).toLocaleDateString();
+            return (
+              <PublicationListItemComponent
+                key={publication.id}
+                primaryComponent={
+                  <MuiLink component={Link} to={`/publication/${publicationId}/public`}>
+                    {publication.title ?? publication.title}
+                  </MuiLink>
+                }
+                secondaryComponent={
+                  <Typography component="span">
+                    {displayDate && <div>{displayDate}</div>}
+                    {publication.contributor &&
```

Comment:
er det ikke types på dette?
```suggestion
                    {publication.contributors &&
```

Reply:
nei, det feltet heter contributor